.github/workflows/ci.yaml

```diff
@@ -67,8 +67,24 @@ jobs:
         cd functions/go
         make check-licenses
 
+  verify-ci:
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: /home/runner/work/kpt-functions-catalog/functions/go
```

Comment:
why do you need GOPATH if it's running python?

I don't feel strongly about this and don't need to take action now: but we should standardizing on one scripting language: currently, we have shell, Go, python, node. It causes and infra complexity.

Reply:
The script now runs `mdrip` which is written in go.

Reply:
I want to get rid of shell in our scripts, since they will become less readable over time.

I can't recall if we have any node scripts.